scripts/backup

```diff
@@ -66,9 +66,9 @@ ynh_backup --src_path="/etc/systemd/system/$app.service"
 #=================================================
 # BACKUP THE DATABASE
 #=================================================
-ynh_print_info --message="Backing up the MySQL database..."
+ynh_print_info --message="Backing up the database..."
 
-ynh_mysql_dump_db --database="$db_name" > db.sql
+ynh_psql_dump_db --database="$db_name" > db.sql
```

Comment:
Just to be sure: the backup before an upgrade is performed using the previous version of the backup script, to it will correctly backup using `ynh_mysql_dump_db` ?